app/views/developers/developers.scala.html

```diff
@@ -101,12 +101,44 @@ <h1 class="flush--top">Developers</h1>
 
     @if(users.length > 0) {
     <div class="form-group>">
-        <p><a class="button" href="mailto:?bcc=@emails">Email @{users.length} developers</a></p>
+        <p>
+            <a class="button" href="mailto:?bcc=@emails" title="Opens external mail client BCC'ing all @{users.length} email addresses.">Email @{users.length} developers</a>
+            <a class="button" href="#" title="Copies all @{users.length} email addresses into the system clipboard." onclick="copyTextToClipboard('@emails'); return false;">Copy to clipboard</a>
```

Comment:
Returning false cancels out the click action (i.e. it prevents following through to the # anchor). See http://stackoverflow.com/questions/128923/whats-the-effect-of-adding-return-false-to-an-click-event-listener for some background